cmd/kube-apiserver/app/options/options.go

```diff
@@ -130,11 +131,11 @@ func gvToMap(gvList string, dest map[string]string) {
 		// "group=group/version". The latter is used when types
 		// move between groups.
 		if !strings.Contains(gv, "=") {
-			dest[apiutil.GetGroup(gv)] = gv
+			dest[apiutil.GetGroup(gv)] = unversioned.GroupVersion{Group: apiutil.GetGroup(gv), Version: apiutil.GetVersion(gv)}
```

Comment:
Sorry I did not understand why cant we use ParseGroupVersion from https://github.com/kubernetes/kubernetes/blob/dae5ac482861382e18b1e7b2943b1b7f333c6a2a/pkg/api/unversioned/group_version.go#L185?